.github/workflows/ci-workflow.yml

```
@@ -75,6 +75,15 @@ jobs:
       - name: Install dependencies
         run: yarn install
 
+      - name: Set Sentry Auth Token
+        run: echo "SENTRY_AUTH_TOKEN=${{ secrets.SENTRY_AUTH_TOKEN }}" >> $GITHUB_ENV
+
+      - name: Build project
+        run: yarn build
+  
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)